config/database.yml.oracle.template

```diff
@@ -1,15 +1,15 @@
 defaults: &defaults
   adapter: oracle_enhanced
-  database: 10.114.22.3:1521/xe
-  encoding: utf8
+  username: nucore_open_development
+  password: password
+  database: localhost:1521/xe
 
 development:
   <<: *defaults
```

Comment:
Did we ever figure out why the `defaults` inheritance wasn't working? On mine I just changed it to not use inheritance at all. That's my only concern really.